project/Version.scala

```diff
@@ -15,7 +15,7 @@
  */
 
 object Version {
-  val geotrellis  = "2.0.0" + Environment.versionSuffix
+  val geotrellis  = "2.0.0-M2"
```

Comment:
I believe it's a typo

Reply:
Also: you should consider creating a custom sbt key called `versionSuffix`, and then you can just change that on the command line **or** in the sbt console, instead of relying on an environment variable.